tests/core/test_memorygrowth_3.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <assert.h>
+#include "emscripten.h"
+
+int get_TOTAL_MEMORY() {
+  return EM_ASM_INT_V({ return TOTAL_MEMORY });
+}
+
+typedef void* voidStar;
+
+int main(int argc, char **argv)
+{
+  int totalMemory = get_TOTAL_MEMORY();
+  int chunk = 1024*1024;
+  volatile voidStar alloc;
```

Comment:
Just so the optimizer doesn't optimize out the allocation entirely (we allocate, but don't really use it).